README.md

```diff
@@ -55,11 +55,21 @@ import VueSelector from 'testcafe-vue-selectors';
 const todoInput = VueSelector('todo-input');
 ```
 
+To obtain a component based vue `ref` value, prepend `:ref` value to the component name and pass it to the VueSelector constructor.
+
+```js
+import VueSelector from 'testcafe-vue-selectors';
+
+const todoInput = VueSelector('ref-todo-item-1:todo-item');
+```
+
 To obtain a nested component, you can use a combined selector.
 ```js
 import VueSelector from 'testcafe-vue-selectors';
 
-const todoItem = VueSelector('todo-list todo-item');
+const todoItem       = VueSelector('todo-list todo-item');
+const todoItem1      = VueSelector('todo-list ref-todo-item-2:todo-item')
+const todoList1Item1 = VueSelector('ref-todo-list-1:todo-list ref-todo-item-1:todo-item')
```

Comment:
`VueSelector` will allow you to search components by the component tag name or the `ref` attribute value.
The `ref` prefix is necessary to distinguish the 'ref' value from a component tag name.
In your implementation, it's not clear what is the first part of the selector string until the `:` character is `ref`.
 
I suggest you implement logic as it was described above:
```js
VueSelector('ref:todo-item')
```
At present, I guess we don't need to implement a possibility to use the AND condition for `ref` and component tag names. I suggest you remove the code related to AND condition support. After that, please create a separate issue where we will discuss it.